c-lightning/2021-08-09-developer-call.md

```diff
@@ -0,0 +1,195 @@
+
+It is worth mentioning that the dual funding is it is only in the open protocol. Once you have the channel established all of that goes away anyway, it is just who is able to open channels to you using it. That being said, I am assuming most people will want to update to the latest release anyway since it is the one that finally enables you to start providing your own liquidity to peers. It has got the tooling for it now.
+
+Does it require a closing of the dual funded channel? And then a reopening?
+
+No. 
+
+If you want them to give you liquidity and you already have a channel with them you would need to close your current channel and then have them reopen a channel where they put liquidity into it.
+
+Yes, we only support a single live channel with a peer at the moment. That’s independent of anything else.
+
+For future releases, when we have splicing in theory it might be possible to renegotiate something with a splice instead of a close and reopen. 
+
+That’s the bit ACINQ really want, they want splicing really badly. Very excited about this release, recommended upgrade for everyone. It has been a long release cycle, much longer than normal. I am hoping the next one will be shorter.
+
+I was curious about limiting the max HTLC size. Why would you want to do that? When I open channels it is so I can route as large HTLCs as possible. 
+
+Because of the escape hole. If you pay me for liquidity and I use everything except the reserve, 99 percent of it, one huge HTLC. Then I drop it onchain unilaterally and the HTLC times out in 3 days. I can get that back in 3 days because the HTLCs are not encumbered with the lease timeout. Normally the deal is you’ve paid me for liquidity, I can’t get my money back for a month. But there is a hole and that is the HTLCs. If you limit the number of HTLCs in flight at one time you reduce the size of that hole that I can get back. Also using up all your liquidity simultaneously at once is not the real case you are going for usually. You probably want this liquidity to last some period of time. These days with MPP, even if they can only use part of the liquidity they can use something else as well. Usually you are opening more liquidity than you expect from a single payment although maybe you’re not. Maybe you are only after that one payment and that’s why you need the liquidity, in which case perhaps that’s fine. In that case you don’t care if they keep it for a month or not. The main reason is that the HTLC is not encumbered by the extra delay that you’ve agreed on on the lease.
```

Comment:
```suggestion
Because of the escape hole. If you pay me for liquidity and I use everything except the reserve, 99 percent of it, one huge HTLC. Then I drop it onchain unilaterally and the HTLC times out in 3 days. I can get that back in 3 days because the HTLCs are not encumbered with the lease timeout. Normally the deal is you’ve paid me for liquidity, I can’t get my money back for a month. But there is a hole and that is the HTLCs. If you limit the number of HTLCs in flight at one time you reduce the size of that hole that I can get back. Also using up all your liquidity simultaneously at once is not the real case you are going for usually. You probably want this liquidity to last some period of time. These days with MPP, even if they can only use part of the liquidity they can use something else as well. Usually you are opening more liquidity than you expect from a single payment although maybe you’re not. Maybe you are only after that one payment and that’s why you need the liquidity, in which case perhaps that’s fine. In that case you don’t care if they keep it for a month or not. The main reason is that the HTLC is not encumbered by the extra delay that you’ve agreed on the lease.
```

Reply:
Done